webpack/examples.config.js

```diff
@@ -1,34 +1,18 @@
-var path = require('path');
-var webpack = require('webpack');
+const path = require('path');
 
-module.exports = {
-  devtool: 'eval',
-  entry: [
-    'webpack-dev-server/client?http://localhost:3000',
-    'webpack/hot/only-dev-server',
-    path.join(__dirname, '../examples/App.js')
-  ],
-  output: {
-    path: path.join(__dirname, '../examples/'),
-    filename: 'bundle.js',
-    publicPath: '/static/'
-  },
+const docsConfig = require('./docs.config');
+
+module.exports = Object.assign(
```

Comment:
HMR wasn't properly working on master anyway (it would complain that it'd have to reload the page). Getting rid of that lets me just merge these two configs.